content/open-source/accoutrement.rst

```diff
@@ -0,0 +1,96 @@
+public: yes
```

Comment:
Oh, true. JavaScript -- what a concept!

Let's tackle this separately in https://www.pivotaltracker.com/story/show/143065833